### DISCUSSION TITLE
Blurring a non-texture in Malt

### DISCUSSION THREAD

**Discussion by Psi25** [bnpr/Malt#390] [Q&A] [answered]
As far as I know, to blur textures in Malt you can only blur textures using the 3 blur nodes. Is there perhaps a way to blur non-texture stuff in Malt?

**Comment by Kolupsy** [comment, marked as answer]
That would be an interesting idea. But as long as there are no node groups in Malt, this would not really be possible. But I can imagine it being added to Malt in the future.

For now you need to render whatever you want to blur to a texture using the 'Screen Pass' pipeline node and then feed that into your Main pass or into another 'Screen Pass' pipeline node.

**Reply by Psi25** [reply]
I hope it gets added to Malt in the future.

**Reply by Psi25** [reply]
I just resumed my project related to this. How do I feed the Screen Pass to the Main Pass?

**Reply by Psi25** [reply]
Do I put the Screen Pass on the Default Render Layer? Then feed it to the Main Pass?

**Reply by Psi25** [reply]
I might be doing something wrong....

![image](https://user-images.githubusercontent.com/103315144/198047256-628e99af-5157-4ee3-89d8-b792e642ce68.png)

**Reply by pragma37** [reply]
Seems like you are using the Background Material for the blur?
You should use a new material with its own node tree.

In this case, though, you could simply link the PrePass->ease output to the MainPass->ease input and use the Blur node inside the Mesh node tree (instead of the Image node you are currently using).

**Reply by Psi25** [reply]
That explains why...

**Reply by Psi25** [reply]
So the textures looks like it's projected into the mesh. I put a Camera Data node and plugged the screen uv, then it works fine now.
